src/store/useColorManagement.tsx

```diff
@@ -0,0 +1,19 @@
+import { useTheme } from "@chakra-ui/react"
+
+/**
+ * Here we can manange colors
+ * of multiple texts and background
+ * from different files
+ * @returns
+ */
+const useColorManagement = () => {
+  const theme = useTheme()
+  let loadingBackgroundColor = theme.colors.primary.main
+
+  return {
+    loadingBackgroundColor
+  }
+
+}
+
+export default useColorManagement;
```

Comment:
This is unnecessary. you are doing again what useTheme is doing.   There is no need for this. If you want an extra property in the useTheme, you can extend the theme, but not sure if is necessary. We keep it with primary color main.